hw/top_darjeeling/data/xbar_dbg.hjson

```diff
@@ -17,18 +17,31 @@
       xbar:       "true",
       pipeline:   false,
     },
+    { name:      "rv_dm.dbg",
+      type:      "device",
+      clock:     "clk_dbg_i",
+      reset:     "rst_dbg_ni",
+      pipeline:  false,
+    },
     { name:       "mbx_jtag.soc",
       type:       "device",
-      addr_space: "soc_dbg",
       clock:      "clk_dbg_i",
       reset:      "rst_dbg_ni",
       pipeline:   false,
     },
+    { name:      "lc_ctrl.dbg",
+      type:      "device",
+      clock:     "clk_peri_i",
+      reset:     "rst_peri_ni",
+      pipeline:  false
+    }
```

Comment:
@Razer6 @neeraj-rv FYI - this connects all three JTAG accessible blocks to the same DBG crossbar.